string trial_name
uint32 start_time
uint32 stop_time